rllib/utils/test_utils.py

```diff
@@ -1362,7 +1362,11 @@ def run_rllib_example_script_experiment(
         args.as_test = True
 
     # Initialize Ray.
-    ray.init(num_cpus=args.num_cpus or None, local_mode=args.local_mode)
+    ray.init(
```

Comment:
Added reinit error ignore:
In case one calls this utility function twice in an example script.